internal/http/services/owncloud/ocs/handlers/cloud/capabilities/capabilities.go

```diff
@@ -116,6 +116,9 @@ func (h *Handler) Init(c *config.Config) {
 		h.c.Capabilities.Dav.Reports = []string{"search-files"}
 	}
 
+	// enable spaces by default
+	h.c.Capabilities.Dav.Spaces = true
```

Comment:
@refs in case we need to disable this functionality how we'll able to do so? Would not be always hardcoded to true?

Reply:
Permanently hardcoded to `true` unless we tackle the capabilities non-hardcoded values. Should I wire to a value from a config file instead? Perhaps that actually causes less troubles 🤔